Processes.md

```diff
@@ -0,0 +1,62 @@
+**Maintain and update processes**
+A few months ago, I walked into a restaurant with a friend to grab a meal. 
+The first thing we needed was a menu(to enable us place our orders) which was supposed to be brought by the waiter, 
+that process ended up taking over 30 min. As we were was waiting, I noticed some customers walk out without placing 
+their orders, their faces displaying disappointment. I wondered how many customers they lost on a daily basis and the 
+amount of money they lost on the orders not made because of the inefficiency at the menu provision and order stage.
+ 
+Recently when I was working on Elixir processes, I remembered the restaurant incident and I thought, 
+processes can be used to hold state. Therefore, a restaurant menu can always be availed to customers on their table
+ (perhaps using a screen on the table), with the new delicacies/meals of the day list. The customers would then be able
+to place orders directly to the kitchen, which would take way much less time. This would mean that the menu list has to 
+be flawless and everything listed on it should be available and the restaurant would have a way of tracking orders made
```

Comment:
This would mean that the menu list has to be up to date with that exists in the kitchen stock inventory with every order updating the state appropriately.